File: example/javamodule/4-test-deps/baz/test/src/baz/BazTests.java
package baz;

import org.junit.Test;
import com.google.common.math.IntMath;

import static org.junit.Assert.*;

public class BazTests {

  @Test
  public void simple() {
    BazTestUtils.bazAssertEquals(Baz.getValue(), IntMath.mean(122, 124));
  }
}

File: example/kotlinlib/testing/2-test-deps/baz/test/src/baz/BazTests.kt
package baz

import com.google.common.math.IntMath
import io.kotest.core.spec.style.FunSpec

class BazTests :
    FunSpec({

        test("simple") {
            BazTestUtils.bazAssertEquals(Baz.VALUE, IntMath.mean(122, 124))
        }
    })
